src/styles/SelectedRevsStyle.ts

```diff
@@ -52,19 +52,17 @@ export const SelectRevsStyles = (mode: string) => {
       },
     },
 
-    listItemButton: {
-      cursor: 'auto !important',
-      padding: '0 !important',
-      alignItems: 'flex-start !important',
+    selectedRevision: {
+      display: 'flex',
+      cursor: 'auto',
+      padding: '0',
+      alignItems: 'flex-start',
```

Comment:
you don't need `cursor` `padding` and `alignItems`, these values are the default values for these properties. It was necessary before to override the button's css properties.

Reply:
Oh okay I see that now